tests/unit_tests/testing/test_sim_telarray_metadata.py

```diff
@@ -17,21 +17,11 @@
 )
 
 
-@pytest.fixture
-def test_metadata_file():
-    return (
-        "tests/resources/"
-        "run000010_gamma_za20deg_azm000deg_North_test_layout_6.0.0"
-        "_test-production-North.simtel.zst"
-    )
-
-
-@pytest.fixture
-def test_metadata(test_metadata_file):
-    return read_sim_telarray_metadata(test_metadata_file)
+def test_metadata(sim_telarray_file_gamma):
```

Comment:
Is this the right file for this test? The rest of the file tests a different module. If anything, it should probably move to the test file above (`test_simtel_io_metdata.py`) and make its name clearer like `test_read_sim_telarray_metadata_type`. Though if you ask me, I would just remove it completely, there's a test already that this function returns two values.